eth/protocols/eth/handshake.go

```diff
@@ -72,7 +72,7 @@ func (p *Peer) Handshake(network uint64, td *big.Int, head common.Hash, genesis
 	}
 	p.td, p.head = status.TD, status.Head
 
-	if p.version >= ETH67 {
```

Comment:
it could have trouble, if remote is using v1.3.x and ETH67,  the handshake would get stuck here.